### DISCUSSION TITLE
Create a relation with the same collection type using the same field name

### DISCUSSION THREAD

**Discussion by cgonzalezsan** [strapi/strapi#8513] [Legacy Topics]
Hi,

I’m trying to set up a relation of a collection type (Articles) with the same collection type (Articles). I would like to have the `articles` relation name but if I use the same name on both ends of the relation, it will say that “_This value is already used._“. Does anyone know how to achieve this or if it's possible?

Please note that I need to have the same field on boths sides of the relation, not different ones (I know that's possible). Also the relation needs to be visible on sides, not "has many" won't work either.

Thanks

<img width="924" alt="Screenshot 2020-10-29 at 17 39 52" src="https://user-images.githubusercontent.com/6850051/97604430-c5c74500-1a0d-11eb-9456-2e9ef2918c6b.png">

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
